src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs

```diff
@@ -71,8 +71,8 @@ public static class OnnxCatalog
         => new OnnxScoringEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnNames, inputColumnNames, modelFile, gpuDeviceId, fallbackToCpu);
 
         /// <summary>
-        /// Creates a new instance of <see cref="DnnImageFeaturizerEstimator"/> which applies one of the pre-trained DNN models in
-        /// <see cref="DnnImageModelSelector"/>to featurize an image.
+        /// Create <see cref="DnnImageFeaturizerEstimator"/>, which applies one of the pre-trained DNN models in
+        /// <see cref="DnnImageModelSelector"/> to featurize an image.
         /// </summary>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="outputColumnName">The name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
```

Comment:
>outputColumnName [](start = 25, length = 16)

type.

Reply:
add data type information


---
In reply to: [277200807](https://github.com/dotnet/machinelearning/pull/3456#discussion_r277200807) [](ancestors = 277200807)